TTHAnalysis/python/plotter/ttH-multilepton/mca-2lss-mcdata-frdata-CP.txt

```diff
@@ -1,41 +1,41 @@
-incl_sigprompt : + ; IncludeMca="ttH-multilepton/mca-includes/mca-2lss-CP-splitdecays.txt"
```

Comment:
I argue that in order to be consistent the *-CP* txt files should be left as they are. The changes should be better applied to new *-diff* txt files :)